tchannel-core/src/main/java/com/uber/tchannel/tracing/Tracing.java

```diff
@@ -50,28 +53,43 @@
 
     private Tracing() {}
 
-    public static void startOutboundSpan(
-            OutRequest<Response> outRequest,
-            Tracer tracer,
-            TracingContext tracingContext
-    ) {
-        if (tracer == null) {
+    /**
+     * @throws RuntimeException
+     *     if the outbound request should fail immediately
+     */
+    public static <V extends Response> void startOutboundSpan(
+            @NotNull OutRequest<V> outRequest,
```

Comment:
I don't think you need `<V extends Response>` in the method, just `OutRequest<? extends Response> outRequest`, since you don't use `V` in any way

Reply:
Without the type parameter, the code (in `SubChannel`) has to resort to raw types. I'll update the calling site in another diff, as a part of the larger clean-up effort.